inference-engine/src/gna_plugin/runtime/pwl.h

```diff
@@ -22,6 +22,7 @@
 #define IDENTITY_DOMAIN 10.0f
 #define PWL_MAX_ERR_PERCENT 1.0f
 #define PWL_MAX_ITERATIONS 2000
+#define PWL_MAX_ITERATIONS_LOG 5000
```

Comment:
While GNA_PWL_MAX_ERROR_PERCENT option is used performance can be affected only if this parameter has small value, without this change the algorithm just won't converge for this value of error.
If we remove GNA_PWL_MAX_ERROR_PERCENT option and will search for the appropriate number of segments for every activation individually, increase of iterations number can increase loading time since the algorithm will converge with the larger number of segments. The branch with pwl search algorithm which we tested for performance contained this change (the increased number of iterations for Log) too.